symplyphysics/laws/electricity/radius_of_a_charged_particle_in_a_magnetic_field.py

```diff
@@ -0,0 +1,46 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Let an arbitrary particle move in a magnetic field around a circle. Then radius of its motion depends on
+## magnetic induction and on velocity, charge and mass of the particle.
+
+## Law is: r = m * v / (q * B), where
```

Comment:
Do we really need this law. You simply substituted `T` for `2pi * r / v` in https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/electricity/period_of_a_charged_particle_in_a_magnetic_field.py

Reply:
> Do we really need this law. You simply substituted `T` for `2pi * r / v` in

I don't quite understand the remark.
Yes, these laws are related, but they give different values and have a similar, but not identical appearance.

I will also give examples of laws from the repository in which the similarity with each other is either the same or higher:
1) https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/waves/wavespeed_from_medium.py

https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/waves/wavespeed_from_medium_permittivity_permeability.py

2) https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/waves/photon_momentum_is_proportional_to_frequency.py

https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/waves/photon_momentum_is_proportional_to_propagation_vec.py

Reply:
Yes, sometimes we add very closely related laws. But for a reason: they are widely used in this form; or it was a practice task. If you think this form is better for some applications, or better known for some users, it's ok to keep it.
Anyway, I expect you to derive one form from another, if you decide to keep it.

Reply:
> Yes, sometimes we add very closely related laws. But for a reason: they are widely used in this form; or it was a practice task. If you think this form is better for some applications, or better known for some users, it's ok to keep it. Anyway, I expect you to derive one form from another, if you decide to keep it.

Ok, got it. I'll add a proof.